src/helpers/loadOrCreateEsaLevel.ts

```diff
@@ -18,7 +18,7 @@ export function loadOrCreateEsaLevel(
   if (!_esaLevel) {
     _esaLevel = new ESALevel(id);
     _esaLevel.universe = universeAddress.toHex();
-    _esaLevel.address = accountAddress;
+    _esaLevel.accountAddress = accountAddress;
```

Comment:
this should be reflected in the idTemplate, the mapping file and anywhere else it is used